setup.py

```diff
@@ -88,14 +105,10 @@
         "snowflake.snowpark._internal",
         "snowflake.snowpark._internal.analyzer",
         "snowflake.snowpark._internal.compiler",
+        "snowflake.snowpark._internal.proto",
         "snowflake.snowpark.mock",
         "snowflake.snowpark.modin",
         "snowflake.snowpark.modin.config",
-        "snowflake.snowpark.modin.core.dataframe.algebra.default2pandas",
```

Comment:
@sfc-gh-azhan can you take a look at these.

Reply:
Those changes look wrong to me.